benchmark/benchmark.py

```diff
@@ -125,6 +125,13 @@ def cli():
         default="default.qubit",
         help="comma-separated list of devices to run the benchmark on (default: %(default)s)",
     )
+    parser.add_argument(
+        "-q",
+        "--qnode",
+        type=lambda x: x.split(","),
+        default="QNode",
```

Comment:
What subclass does `QNode` correspond to? Or does this refer to the `qml.QNode` function (so default is to pick the 'best' QNode for the device)?

Reply:
yep, the latter. Essentially, no choice was being allowed before, so the default here is just what the benchmarks were previously doing